test/catalyst/assetAttributesRegistry/fixtures.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  ethers,
+  deployments,
+  getNamedAccounts,
+  getUnnamedAccounts,
+} from 'hardhat';
+import {BigNumber, Contract, Event} from 'ethers';
+import {Block} from '@ethersproject/providers';
+import {waitFor} from '../../utils';
+import {transferSand} from '../utils';
+import {setupGemsAndCatalysts} from '../gemsCatalystsRegistry/fixtures';
+
+export const setupAssetAttributesRegistry = deployments.createFixture(
+  async () => {
+    await deployments.fixture();
+    await setupGemsAndCatalysts();
```

Comment:
Not sure if we still want this dependency on other fixture script

Reply:
fixed.